### PR TITLE
Create mediawiki.yml

### PR DESCRIPTION
**Pull request by waldyrious**

### CODE REVIEW

**Comment by suriyaa** [comment]
👍

File: _data/projects/mediawiki.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://www.mediawiki.org
+tags:
+- mediawiki
+- wiki
+- web
+- cms
+- php
+upforgrabs:
+  name: Easy
+  link: https://phabricator.wikimedia.org/tag/easy/
```
Comment (shiftkey) [inline _data/projects/mediawiki.yml:12]: Would it be better to navigate the user straight to the task list? e.g. https://phabricator.wikimedia.org/maniphest/query/PaVmrjYb6zFl/#R

Comment (waldyrious) [reply _data/projects/mediawiki.yml:12]: I thought about that (I didn't like linking to a landing page either), but I'm not sure how permanent that link is. Should I change?

Comment (shiftkey) [reply _data/projects/mediawiki.yml:12]: This is fine - "Open tasks" is available in the sidebar, it's not like it's hard to find...

**Pull request merged**